src/ReconNess.Web/Controllers/TargetsController.cs

```diff
@@ -461,5 +461,54 @@ public async Task<IActionResult> GetDashboard([FromRoute] string targetName, Can
 
             return Ok(await this.targetService.GetDashboardAsync(targetName, cancellationToken));
         }
+
+        /// <summary>
+        /// Export the target with all the rootdomains.
+        /// </summary>
+        /// <remarks>
+        /// Sample request:
+        ///
+        ///     GET api/targets/export/{targetName}/alldata
+        ///
+        /// </remarks>
+        /// <param name="targetName">The target name</param>
+        /// <param name="cancellationToken">Notification that operations should be canceled</param>
+        /// <returns>The json file with the target and the rootdomain data</returns>
+        /// <response code="200">Returns the json file with the rootdomain and the subdomains data</response>
+        /// <response code="400">Bad Request</response>
+        /// <response code="401">If the user is not authenticate</response>
+        /// <response code="404">Not Found</response>
+        [HttpPost("export/alldata/{targetName}")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public async Task<IActionResult> ExportTargetInfo([FromRoute] string targetName, CancellationToken cancellationToken)
```

Comment:
we don't need this new endpoint we have the endpoint 'export' if that endpoint is missing data we need to provide that there

https://github.com/reconness/reconness/blob/8e9378f85e7f4cdca14a7560a8879b0b655cc141/src/ReconNess.Web/Controllers/TargetsController.cs#L405